app/controllers/AuthController.scala

```diff
@@ -62,7 +62,7 @@ class AuthController @Inject() (
           }
         case None =>
           sessionManager.retrieveRequestToken(
-            proxyRedirect(routes.AuthController.authenticate()) + s"?redirect=${getRedirectURL(request, redirect)}"
```

Comment:
This one I cannot claim to have full read on. As a scala 'user', not an expert, I understand that parameter-less functions can be referenced/invoked without parentheses. Though can != should so I am not quite sure why a framework bump would enforce this. 

I was getting the compilation error `authenticate does not take parameters`.